client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import SingleAppContainer from './SingleAppContainer.js';
+import {topFree} from './mockData.js';
+import {topGrossing} from './mockData.js';
+import {topPaid} from './mockData.js';
+import {trending} from './mockData.js';
+import './TopChartsStyle.css';
+import '../styles.css';
+ 
+class TopCharts extends Component {
+  render() {
+    const makeColumn = item => (
+        <div key={item.uniqueID}>
+            <SingleAppContainer
+            listNum={item.listNum}
+            url={item.url}
+            title={item.title}
+            category={item.category}
+            rating={item.rating}
+            />
+        </div>
+    );
+
+    return (
+      <div className="col-12 bg-light mx-auto text-center">
+        <div class="section-header">
+          <h4>Top Charts</h4>
+          <h8>For Wear OS</h8>
+        </div>
+ 
+        <div class="col-25">
```

Comment:
we should probably not hardcode this to 25% and always expect it to be four columns. what if we want to create a top charts cluster with 3 charts or 2 charts? then we would also need to update the frontend. Take a look at flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/